.gitignore

```diff
@@ -1,3 +1,4 @@
 /.idea
 /vendor
 /composer.phar
+/templates/php-component/vendor/
```

Comment:
muzes nechat .gitignore u sablony, prepise to tady ten spolecny
stejne si cim dal vic myslim, ze ten common zrusime - tech spolecnych veci je strasne malo